src/instructlab/configuration.py

```diff
@@ -249,6 +249,7 @@ class _serve_vllm(BaseModel):
 
     # arguments to pass into vllm process
     vllm_args: list[str] | None = None
+    background: bool = True
```

Comment:
```suggestion
    enable_serving_output: bool = False
```

To match the CLI flag?